### ISSUE TITLE
Current stage optimizations

### ISSUE THREAD

**Issue by ret7020** [ITClassDev/FrontEnd#3]
## About
React scripts default builder(webpack warning).
![2023-01-08_11-17](https://user-images.githubusercontent.com/55328925/211187743-aef3a32c-5144-48cc-b21b-8a7c44849f69.png)
It is not a big problem now, but we cannot leave this unnoticed, as we plan to make the system as productive as possible. Therefore, this is an important issue that needs to be fixed now.

## Tech info
Building commit: 4f422023e65ec6b3dbf8c447e849916eac18e6e9
OS: Linux

**Comment by ret7020** [comment]
```
 "dependencies": {
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "antd": "^5.0.3",
    "axios": "^1.2.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-router-dom": "^6.4.3",
    "react-scripts": "5.0.1",
    "react-syntax-highlighter": "^15.5.0",
    "web-vitals": "^2.1.4"
  },
```
But I would like to note that we have very few dependencies, there is no musr, and that's cool!

**Comment by ret7020** [comment]
After deleting obsolete/legacy files with code
![image](https://user-images.githubusercontent.com/55328925/211187904-32735ef9-4cab-42a5-bb15-ebfbfde47507.png)

**Comment by ret7020** [comment]
![image](https://user-images.githubusercontent.com/55328925/211189582-d93401f6-04e8-43f9-a22b-aab83fdaf532.png)
So now, we are using lazy loading via React.Suspense and React.lazy on react router each children (<Outlet/> rendering) page.
But think about API docs page, it will have a lot content and we have to optimize it in future.
Issue fixed in c564235ada7ffd91d83aa78bbd51afb9e8d571b4
